test/scraping.js

```diff
@@ -139,7 +139,37 @@ describe('scraping', function() {
 			.then(function(results) {
 				var authors = results.authors;
 				assert.deepEqual(JSON.stringify(authors), expectedAuthors);
-			})
+			});
+		});
+	});
+
+	it('should get basic EPrints metadata', function() {
+		url = 'http://eprints.gla.ac.uk/113711/';
+		return preq.get(url).then(function(callRes) {
+			var chtml = cheerio.load(callRes.body);
+
+			return meta.parseEprints(chtml)
+			.then(function(results) {
+				['eprintid', 'datestamp', 'title', 'abstract', 'issn', 'creators_name', 'publication', 'citation'].forEach(function(key) {
+					if(!results[key]) {
+						throw new Error('Expected to find the ' + key + ' key in the response!');
+					}
+				});
+			});
+		});
+	});
+
+	it('should get EPrints author tags correctly', function() {
```

Comment:
Since you're using the same url here for both tests, I wouldn't make two different requests for the same info. You can combine the tests into one and have two assert statements in it. (Yes, I realise there are some tests above that do this, I let it slip by me, whoops :P)